shell/app-shell/app-shell.js

```diff
@@ -372,12 +378,14 @@ class AppShell extends Xen.Base {
       state.userName = state.user.name;
     }
     if (state.config && state.key && state.user) {
-      // modifying config only matters before initializing arc-host
       let isProfile = state.user.profiles && state.user.profiles[state.key];
+      /*
```

Comment:
Should the commented out block be removed?

Reply:
Same as above.